include/fmt/std.h

```diff
@@ -28,6 +28,18 @@ void write_escaped_path(basic_memory_buffer<Char>& quoted,
                         const std::filesystem::path& p) {
   write_escaped_string<Char>(std::back_inserter(quoted), p.string<Char>());
 }
+#  ifdef _WIN32
+template <>
+void write_escaped_path<char>(basic_memory_buffer<char>& quoted,
+                              const std::filesystem::path& p) {
+  auto s = p.u8string();
+  // This is safe reinterpret_cast:
+  // https://stackoverflow.com/a/57453713/590688
```

Comment:
Let's remove this comment cast to `const char*` is obviously fine.

Reply:
Done